opm/output/eclipse/ActiveIndexByColumns.cpp

```diff
@@ -85,10 +85,10 @@ namespace {
         const auto& [outer, middle] = inferOuterLoopOrdering(cartDims);
 
         std::transform(colGlobIx.begin(), colGlobIx.end(), colGlobIx.begin(),
-                       [&cartDims, &getIJK, outer=outer, middle=middle]
+                       [&cartDims, &getIJK, Outer=outer, Middle=middle]
```

Comment:
Why not just
```
[&cartDims, &getIJK, outer, middle]
```

Reply:
because capturing structured bindings directly requires c++-20

Reply:
Right! Thanks.